ATL/schema.sql

```diff
@@ -0,0 +1,69 @@
+Create Table Passengers(
+Id int identity(1,1) Primary key,

```

Comment:
At the start of the problem there is a statement about the number of passengers,
Located in Atlanta, Georgia in the United States, ATL served 93.7 million passengers in 2022
According to my calculations, we will reach the max value of an integer in ~22 years(2147483647/100000000). It might be worth looking into using a BIGINT.